Dockerfile

```diff
@@ -6,5 +6,7 @@ RUN pip install -e .
 RUN pip install https://github.com/rogerbinns/apsw/releases/download/3.38.1-r1/apsw-3.38.1-r1.zip \
   --global-option=fetch --global-option=--version --global-option=3.38.1 --global-option=--all \
   --global-option=build --global-option=--enable-all-extensions
-CMD ["uvicorn", "dj.api.main:app", "--host", "0.0.0.0", "--port", "8000", "--reload"]
+
+RUN pip install jupyter nb-black
```

Comment:
Do we want to do this here or keep track of these in a separate requirements file?

Reply:
Let me try moving it to the test requirements. I believe the only place it's used is in the pre-commit runs to make sure the example notebooks are all formatted nicely.

Reply:
Actually, pre-commit handles it's own requirements, I just went ahead and removed this.